<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>411</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sentinel-2-Indices berechnen</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>491</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sentinel-2 Indices über die Spektralbänder berechnen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_calc">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="s1_pfad">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="s2_pfad">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>150</y>
     <width>261</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pfad zum Spektralband 1 (S1)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>191</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Pfad zum Spektralband 2 (S2)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Der Index wird nachfolgend als (S1-S2)/(S1+S2) berechnet. </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>270</y>
     <width>511</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Ein Tool für die Berechnung aus einer Liste von benannten Indices ist zurzeit in Arbeit.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>631</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Immer Spektralbänder gleicher Auflösung nutzen! Für einen Überblick über die Indices siehe Dokumentation.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_such1">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_such2">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Suchen...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>310</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Schließen</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
